src/sonic-yang-models/yang-models/sonic-copp.yang

```diff
@@ -0,0 +1,243 @@
+module sonic-copp {
+
+	yang-version 1.1;
+
+	namespace "http://github.com/Azure/sonic-copp";
+	prefix copp;
+
+	description "CoPP YANG Module for SONiC OS";
+
+	revision 2021-03-31 {
+		description
+			"First Revision";
+	}
+
+	container sonic-copp {
+		container COPP_GROUP {
+			list COPP_GROUP_LIST {
+				key name;
+
+				leaf name {
+					type string;
+					description
+						"Control plane policing group name";
+				}
+
+				leaf queue {
+					type uint8 {
+						range "0..47" {
+							error-message "Invalid value passed for queue";
+						}
+					}
+					description
+						"CPU queue id";
+				}
+
+				leaf trap_priority {
+					type uint16 {
+						range "0..1023" {
```

Comment:
From the SAI headers the range is dependent on platforms. I am not sure why the priority is restricted to 0-1023
     *
     * @type sai_uint32_t
     * @flags CREATE_AND_SET
     * @default attrvalue SAI_SWITCH_ATTR_ACL_ENTRY_MINIMUM_PRIORITY
     * @validonly SAI_HOSTIF_TRAP_ATTR_PACKET_ACTION == SAI_PACKET_ACTION_TRAP or SAI_HOSTIF_TRAP_ATTR_PACKET_ACTION == SAI_PACKET_ACTION_COPY
     */
    SAI_HOSTIF_TRAP_ATTR_TRAP_PRIORITY

Reply:
Removed `range "0..1023" {`